win32/src/win32gui.i

```diff
@@ -1698,24 +1698,23 @@ static PyObject *PySetString(PyObject *self, PyObject *args)
 static PyObject *PySetMemory(PyObject *self, PyObject *args)
 {
 	void *addr;
-	const void *src;
 	PyObject *obaddr, *obsrc;
-	Py_ssize_t nbytes;
 
-	// @pyparm int|addr||Address of the memory to reference 
+	// @pyparm int|addr||Address of the memory to reference
 	// @pyparm string or buffer|String||The string to copy
 	if (!PyArg_ParseTuple(args, "OO:PySetMemory", &obaddr, &obsrc))
 		return NULL;
 	if (!PyWinLong_AsVoidPtr(obaddr, &addr))
 		return NULL;
-	if (PyObject_AsReadBuffer(obsrc, &src, &nbytes)==-1)
-		return NULL;
-	if (IsBadWritePtr(addr, nbytes)) {
+    BufferView pybuf(obsrc);
```

Comment:
ditto here (I know this file already has some mixed tabs and spaces, but we should try and avoid making that worse)

Reply:
solved by updated PR